cluster-autoscaler/core/scaledown/planner/planner.go

```diff
@@ -97,20 +99,26 @@ func New(context *context.AutoscalingContext, processors *processors.Autoscaling
 // up-to-date information about the cluster.
 // Planner will evaluate scaleDownCandidates in the order provided here.
 func (p *Planner) UpdateClusterState(podDestinations, scaleDownCandidates []*apiv1.Node, as scaledown.ActuationStatus, pdb []*policyv1.PodDisruptionBudget, currentTime time.Time) errors.AutoscalerError {
+	var err error
 	p.latestUpdate = currentTime
 	p.actuationStatus = as
+	p.pdbRemainingDisruptions, err = pdbdisruptions.NewPdbRemainingDisruptions(pdb)
+	if err != nil {
+		p.CleanUpUnneededNodes()
+		return errors.ToAutoscalerError(errors.UnexpectedScaleDownStateError, err)
```

Comment:
I think this should just be internal error. Malformed PDB doesn't match the description of `UnexpectedScaleDownStateError`.